### DISCUSSION TITLE
Can you migrate an existing Org, built from ALZ, to org-formation?

### DISCUSSION THREAD

**Discussion by schwinbp** [org-formation/org-formation-cli#551] [Q&A]
Hi,
 We have an existing organization consisting of several OUs and 300+ accounts that we manage through AWS Landing Zone. Now that ALZ is EOL, we were trying to migrate to AWS Control Tower which is awful. Is it possible to migrate an existing ALZ org over to org-formation? Is there any docs on how to do this?
Brian

**Comment by OlafConijn** [comment]
hi @schwinbp , thanks for asking!

to start managing any organization using org-formation you can run either the [init](https://github.com/org-formation/org-formation-cli/blob/master/docs/cli-reference.md#org-formation-init) or [init-pipeline](https://github.com/org-formation/org-formation-cli/blob/master/docs/cli-reference.md#org-formation-init-pipeline) command.
after having done this you can start using org-formation incrementally, adding "tasks" and "templates" you deploy to your AWS Organization.

its a somewhat "technical" tool, if you need help - feel free to reach out [on slack](https://join.slack.com/t/org-formation/shared_invite/enQtOTA5NjM3Mzc4ODUwLTMxZjYxYzljZTE5YWUzODE2MTNmYjM5NTY5Nzc3MzljNjVlZGQ1ODEzZDgyMWVkMDg3Mzk1ZjQ1ZjM4MDhlOGM)
